src/enrichment_facility.h

```diff
@@ -233,81 +266,126 @@ class EnrichmentFacility : public cyclus::Facility {
 
   cyclus::Material::Ptr Enrich_(cyclus::Material::Ptr mat, double qty);
 
-  ///   @brief records and enrichment with the cyclus::Recorder
-  void RecordEnrichment_(double natural_u, double swu);
+  ///  @brief calculates the feed assay based on the unenriched inventory
+  double FeedAssay();
 
-  #pragma cyclus var {"tooltip": "input commodity", \
-                      "doc": "commodity that the enrichment facility accepts", \
-                      "uitype": "incommodity"}
+  ///  @brief records and enrichment with the cyclus::Recorder
+  void RecordEnrichment_(double natural_u, double swu);
+  
+  #pragma cyclus var {							\
+    "tooltip": "input commodity",					\
+    "doc": "commodity that the enrichment facility accepts",		\
+    "uitype": "incommodity" \
+  }
   std::string in_commod;
-  #pragma cyclus var {"tooltip": "output commodity", \
-                      "doc": "commodity that the enrichment facility supplies", \
-                      "uitype": "outcommodity"}
+  #pragma cyclus var { \
+    "tooltip": "output commodity",					\
+    "doc": "commodity that the enrichment facility supplies",		\
+    "uitype": "outcommodity" \
+  }
   std::string out_commod;
-  #pragma cyclus var {"tooltip": "input commodity recipe", \
-                      "doc": "recipe for enrichment facility's input commodity", \
-                      "uitype": "recipe"}
+  #pragma cyclus var { \
+    "tooltip": "input commodity recipe",				\
+    "doc": "recipe for enrichment facility input commodity",		\
+    "uitype": "recipe" \
+  }
   std::string in_recipe;
-
-  #pragma cyclus var {"default": 0.03, "tooltip": "tails assay", \
-                      "doc": "tails assay from the enrichment process"}
+  #pragma cyclus var { \
+    "tooltip": "tails commodity",					\
+    "doc": "tails commodity supplied by enrichment facility",		\
+    "uitype": "outcommodity" \
+  }
+  std::string tails_commod;
+  #pragma cyclus var { \
+    "default": 0.03, "tooltip": "tails assay",				\
+    "doc": "tails assay from the enrichment process" \
+  }
   double tails_assay;
-  #pragma cyclus var {"default": 1e299, "tooltip": "SWU capacity", \
-                      "doc": "separative work unit (SWU) capcity of " \
-                             "enrichment facility"}
+  #pragma cyclus var { \
+    "default": 1e299,					       \
+    "tooltip": "SWU capacity (kgSWU/month)",			       \
+    "doc": "separative work unit (SWU) capacity of enrichment " \
+           "facility (kgSWU/month) "		       \
+  }
   double swu_capacity;
-  #pragma cyclus var {"default": 1e299, "tooltip": "maximum inventory size", \
-                      "doc": "maximum inventory capacity of natural uranium in " \
-                             "the enrichment facility"}
+  #pragma cyclus var { \
+    "default": 1e299, "tooltip": "max inventory size (kg)",		\
+    "doc": "maximum total inventory of natural uranium in "		\
+           "the enrichment facility (kg)" \
+  }
   double max_inv_size;
```

Comment:
Changed the following variable names:
in_commod -> feed
in_commodity() -> feed_commod()
out_commod -> product
in_recipe -> feed_recipe
initial_reserves -> initial_feed
InitialReserves -> InitialFeed
max_inv_size -> max_feed_inventory

I prefer to keep 'inventory' rather than 'size' because that is the precise word to describe the amount of material stored at the facility.